File: Java/api/treehopper.libraries/src/main/java/io/treehopper/libraries/sensors/inertial/IGyroscope.java
package io.treehopper.libraries.sensors.inertial;

import com.badlogic.gdx.math.Vector3;
import io.treehopper.libraries.sensors.IPollable;

/**
 * Gyroscope interface
 */
public interface IGyroscope extends IPollable {
    Vector3 getGyroscope();
}

File: NET/API/Treehopper.Libraries/Sensors/Inertial/IGyroscope.cs
using System.Numerics;

namespace Treehopper.Libraries.Sensors.Inertial
{
    /// <summary>
    ///     Three-axis gyroscope sensor
    /// </summary>
    public interface IGyroscope : IPollable
    {
        /// <summary>
        ///     The three-axis gyroscope data, in DPS (degrees per second)
        /// </summary>
        Vector3 Gyroscope { get; }
    }
}